pom.xml

```diff
@@ -458,7 +464,7 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-junit-jupiter</artifactId>
-            <version>5.12.0</version>
+            <version>4.0.0</version>
```

Comment:
lets make use of dependency mgmt here as well

Reply:
so should I add v4 in apk.parser and revert back to v5 here right?